csv-res-filter/pom.xml

```diff
@@ -21,7 +21,7 @@
 		<dependency>
 			<groupId>com.ibm.g11n.pipeline</groupId>
 			<artifactId>gp-res-filter</artifactId>
-			<version>1.2.0</version>
+			<version>1.2.6</version>
```

Comment:
User 1.2.7-SNAPSHOT - to match other modules' version.
We replace all of SNAPSHOT version with the actual release version using maven release plugin automatically.

Reply:
updated